src/components/Repos/NotesEditForm.js

```diff
@@ -0,0 +1,44 @@
+import { useParams, useNavigate } from "react-router-dom"
+import { useState, useEffect } from 'react'
+
+function NotesEditForm() {
+  const { username, noteId } = useParams()
+  const [note, setNote] = useState({contents: '', repo: ''})
```

Comment:
Why is this state initialised using only contents and repo, is he fetch request (line 9) not fetching the whole object for the matching ID from your data store?